votingpool/db.go

```diff
@@ -70,7 +139,7 @@ func putPool(tx walletdb.Tx, votingPoolID []byte) error {
 // loadAllSeries returns a map of all the series stored inside a voting pool
 // bucket, keyed by id.
 func loadAllSeries(tx walletdb.Tx, votingPoolID []byte) (map[uint32]*dbSeriesRow, error) {
```

Comment:
After discussing this some more, see #215 for details on the planned direction to resole this in waddrmgr.  I suspect a similar approach here wold work well.  We're also adding a `Cursor` to walletdb, so depending on how the keys are actually stored and the need to do scans, there might be further optimization potential there.  Given the current code though, I suspect an iterator-style callback will work well here too.